faker/providers/internet/__init__.py

```diff
@@ -115,6 +115,13 @@ class Provider(BaseProvider):
         '?txtsize=55&txt={width}x{height}&w={width}&h={height}',
         'https://www.lorempixel.com/{width}/{height}',
         'https://dummyimage.com/{width}x{height}',
+        'https://baconmockup.com/{width}/{height}',
+        'https://loremflickr.com/{width}/{height}',
+        'https://placekitten.com/{width}/{height}',
+        'https://www.placecage.com/{width}/{height}',
```

Comment:
I'm not sure the images on site have used according to their license. In fact, I don't see any mention of licensing or attribution anywhere on this website.

Reply:
Placekitten's license: https://placekitten.com/attribution.html

placeimg's license is creativecommon.